pkg/services/serviceaccounts/secretscan/webhook.go

```diff
@@ -45,7 +66,6 @@ func (wClient *webHookClient) Notify(ctx context.Context,
 	values := map[string]interface{}{
 		"alert_uid":                uuid.NewString(),
 		"title":                    "SecretScan Alert: Grafana Token leaked",
-		"image_url":                "https://images.pexels.com/photos/5119737/pexels-photo-5119737.jpeg?auto=compress&cs=tinysrgb&w=300", //nolint
```

Comment:
Haha, curious why it was here?

Reply:
:smile: it's what happens when things go from hackathon project to project

Reply:
Ok, nice bird :smiley: